HomeWork/src/main/java/arraylist/TasksWithList.java

```diff
@@ -65,20 +69,20 @@ public int countOccurrence(List<String> elements, String value) {
         return counter;
     }
 
-    static public <T> List<T> toList(T[] elements) {
-        return Arrays.asList(elements);
+    public static <T> List<T> toList(T[] elements) {
+        return new ArrayList<>(Arrays.asList(elements));
     }
 
-    static public List<Integer> findUnique(List<Integer> elements) {
-        List<Integer> repeatedDigits = new java.util.ArrayList<>();
+    public static List<Integer> findUnique(List<Integer> elements) {
 
         for (int i = 0; i < elements.size(); i++) {
+            Integer valueI = elements.get(i);
             for (int j = i + 1; j < elements.size(); j++) {
-                if (elements.get(i).equals(elements.get(j))) {
-                    repeatedDigits.add(elements.get(j));
-                    elements.removeAll(repeatedDigits);
-                    j = elements.size();
-                    i--;
+                Integer valueJ = elements.get(j);
+                if (valueI.equals(valueJ)) {
+                    elements.remove(valueJ);
+                    elements.remove(valueI);
+                    j--;
```

Comment:
What are trying to do here? Why don't you use a list as a result, and just check if that list contains a value from the input list or not? Change this part, as it can be done in a much simpler way.